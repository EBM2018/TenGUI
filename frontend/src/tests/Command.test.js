import ActTextCheck from '../react/FishtankAdmin/Command/Command';
import React from 'react';
import renderer from 'react-test-renderer';

test ('derp',()=>{
    expect('True').toBeTruthy;
});
