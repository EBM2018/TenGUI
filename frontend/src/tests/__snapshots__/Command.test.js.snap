// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`does this render 1`] = `
<div>
  <div
    id="title"
  >
     Mon fils rouge 
  </div>
  <div
    id="ButtonLayoutContainer"
  >
    <button
      id="Start"
      onClick={[Function]}
      type="button"
    >
      Ouverture
    </button>
    <button
      id="AskSummary"
      onClick={[Function]}
      type="button"
    >
      Faites un résumé
    </button>
    <button
      id="AskFeedback"
      onClick={[Function]}
      type="button"
    >
      Feedback
    </button>
  </div>
</div>
`;
